OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -21,6 +22,12 @@ public interface ISpriteLoader
 		bool TryParseSprite(Stream s, out ISpriteFrame[] frames);
 	}
 
+	public interface ISpriteLoader2 : ISpriteLoader
```

Comment:
> Let me explain. each building in dune2 split into multiple tiles. do sequences support this?

Yes, in combination with an appropriate render trait (right now we have special cases for building bibs and bridges, but refactoring those into a general tiled sprite rendering trait has been on the todo list for a long time).

> his mean that data from .map file should be put into yaml files. So, why we still need icon.icn then?

This fits in with the existing conventions and best practices that we have in place, yes.  There is no fundamental reason why you couldn't use icon.map from your sequence parser, and have a line in each animation definition referencing the map.  Alternatively, you can do a one-shot conversion to yaml, use the default sequence parser, and not worry about icon.icn again.

> why not to just put everything in newly created .shp files?

We consider converting data from one weird proprietary format to another weird proprietary format to be an inelegant solution.